crates/solver/src/settlement_access_list.rs

```diff
@@ -0,0 +1,104 @@
+use ethcontract::{Address, H256};
+use reqwest::{
+    header::{HeaderMap, HeaderValue},
+    Client, Url,
+};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize)]
+pub struct TenderlyRequest {
```

Comment:
nit: I think there is no harm in deriving `Clone` for all the new structs.

Reply:
Sure, added.